### PR TITLE
Refactor community-testimonial to accept core-image

### PR DESCRIPTION
**Pull request by marcod1419**
#90

This pull requests modifies the existing `community-testimonial` component to accept the TDS `core-image` component rather than using its own `` element. Additionally, image size is now limited to 60x60, and the dependency for `prettier` was updated.

prepr:
```
Changes:
 - @tds/community-testimonial: 1.0.0-preview => 1.0.0
 - @tds/community-toggle-switch: 1.0.0-preview => 1.0.0

lerna info auto-confirmed 

🤔 If this is not what you expected, ensure that your commit messages follow the Conventional Commits specification (https://conventionalcommits.org) and try again.

🚀 Please paste the entire output of this task into the body of your Pull Request so that a maintainer can verify before merging/publishing.
✨  Done in 8.43s.
```

### CODE REVIEW

File: packages/Testimonial/Testimonial.jsx [hunk 1]
```diff
@@ -37,19 +33,16 @@ const Testimonial = ({ testimonial, imgSrc, imgAlt, title, additionalInfo, ...re
 Testimonial.propTypes = {
   /** Testimonial main text */
   testimonial: PropTypes.string.isRequired,
-  /** Image URL - suggested size is 60x60px */
-  imgSrc: PropTypes.string,
+  /** Image Component - suggested & max size is 60x60px. */
+  image: componentWithName('Image'),
```
Comment (theetrain) [inline packages/Testimonial/Testimonial.jsx]: Unfortunately, the airbnb proptype `componentWithName` bloats the final build of this component. We can either figure out how to properly tree shake this method, or use a simplified, custom validator:

```js
/* eslint-disable import/prefer-default-export */

import { warn } from './warn'

export const componentWithName = (componentName, props, propName) => {
  if (props[propName].type.name !== componentName) {
    warn(
      '@tds/community-testimonial',
      "Component supplied to the 'image' prop must be @tds/core-image"
    )
  }
}

```

**Comment by theetrain** [comment]
```Changes:
 - @tds/community-testimonial: 1.0.0-preview => 1.0.0
 - @tds/community-toggle-switch: 1.0.0-preview => 1.0.0
 - @tds/util-proptypes: 1.0.0-preview => 1.0.0 (private)```

File: config/rollup.config.js [hunk 2]
```diff
@@ -34,6 +34,9 @@ export default opts => {
       }),
       commonjs({
         include: '../../node_modules/**',
+        namedExports: {
```
Comment (marcod1419) [inline config/rollup.config.js]: I gotta remove this.

File: packages/Testimonial/package.json [hunk 3]
```diff
@@ -27,12 +27,13 @@
...
-    "@tds/core-image": "^1.0.0",
     "@tds/core-paragraph": "^1.0.2",
     "@tds/core-responsive": "^1.1.0",
     "prop-types": "^15.6.0"
   },
   "devDependencies": {
-    "@tds/shared-safe-rest": "^1.0.0"
+    "@tds/shared-safe-rest": "^1.0.0",
+    "@tds/util-proptypes": "^1.0.0-preview",
+    "airbnb-prop-types": "^2.11.0"
```
Comment (marcod1419) [inline packages/Testimonial/package.json]: Remove!

**Pull request merged**